transit/tests/api_test/reports/test_average_kilometers_per_shipment_by_transporter.py

```diff
@@ -0,0 +1,38 @@
+import pandas as pd
+from django.test import TestCase
+
+from transit.tests.api_test.reports.utils import ReportTestCaseMixin
+
+
+class TestReportCapacity(ReportTestCaseMixin, TestCase):
+    _URL = 'average_kilometers_per_shipment'
+
+    def setUp(self):
+        super(TestReportCapacity, self).setUp()
+        self.partially_complete_shipment.transporter_base_cost = 123.0
```

Comment:
Is transporter base cost required by avg km per shipment report?

Reply:
yes. Number of kilometers are usually added together with 'base_transporter_cost' (as a part of CostForm). And this field is required in form hence this cost is required.